### PR TITLE
Add string properties for IfEndsWith and IfNotEndsWith

### PR DESCRIPTION
**Pull request by dwakel**
#22 
@mantinband Issue 22 complete with all tests passing

### CODE REVIEW

File: src/Validators/Validator.Strings.cs [hunk 1]
```diff
@@ -109,4 +109,22 @@ internal static void ThrowIfLengthNotEquals(string value, string paramName, Exce
             ExceptionThrower.Throw(paramName, exceptionCustomizations, $"String length should be equal to {length}.");
         }
     }
+
+    [MethodImpl(MethodImplOptions.AggressiveInlining)]
+    internal static void ThrowIfEndsWith(string value, string paramName, ExceptionCustomizations? exceptionCustomizations, string str)
+    {
+        if (value.EndsWith(str))
+        {
+            ExceptionThrower.Throw(paramName, exceptionCustomizations, $"String should not end with {str}.");
```
Comment (mantinband) [inline src/Validators/Validator.Strings.cs:118]: Small change to align with the other error messages that include the given string (H&B)
```suggestion
            ExceptionThrower.Throw(paramName, exceptionCustomizations, $"String should not end with '{str}'.");
```

File: tests/UnitTests/ValidatableExtensions/ValidatableExtensions.StringsTests.cs [hunk 2]
```diff
@@ -310,4 +310,60 @@ public void ThrowIfLenghtNotEquals_WhenLengthEquals_ShouldNotThrow()
         // Assert
         action.Should().NotThrow();
     }
+
+    [TestMethod]
+    public void ThrowIfEndsWith_WhenEndsWith_ShouldNotThrow()
```
Comment (mantinband) [inline tests/UnitTests/ValidatableExtensions/ValidatableExtensions.StringsTests.cs:315]: I'm guessing you meant:
```suggestion
    public void ThrowIfEndsWith_WhenNotEndsWith_ShouldNotThrow()
```
?

**Comment by mantinband** [comment]
Looks great, thanks for adding this feature 🔥! You can bump the version in `Throw.csproj` from `1.1.2` to `1.1.3` so that these changes will be auto publish to nuget.org

**Review by mantinband — APPROVED** [review]

**Comment by dwakel** [comment]
> Looks great, thanks for adding this feature 🔥! You can bump the version in `Throw.csproj` from `1.1.2` to `1.1.3` so that these changes will be auto publish to nuget.org

What

**Pull request merged**